integrated_tokenhandler.md

```diff
@@ -1,33 +1,40 @@
 # Integrating token handler with ansible.
 
-To avoid having to consistently check for valid storedsafe token, this can be integrated directly into ansible via a pre_task.
+To avoid having to consistently check for valid storedsafe token, this can be integrated directly into handler using a tokenhandler login script 
 
-## ansible pre task example
+## wrapper.sh 
 
-In a playbook that requires a valid token, or site.yml, add the following:
+Create a simple script that utilizes tokenhandler to check for valid token, and log in if none is found
 
-```
-- hosts: all
-  gather_facts: True
-  pre_tasks:
-    - name: obtain storedsafe toke
-  ansible.builtin.command:
-    cmd: ../../tokenhandler/wrapper.sh
-  delegate_to: localhost
-  become: no
-  run_once: true
-```
-run_once is necessary to only call login once. 
-wrapper.sh is just a simple wrapper:
+For example:
 
 ```
-#!/usr/bin/bash
-# relative to ansible/playbooks/
-TOKENHANDLER=../../tokenhandler/tookenhandler.py 
-${TOKENHANDLER} check > /dev/null
-if [ $? -eq 1 ]; then
-  ${TOKENHANDLER} login
+#!/bin/env bash
+
+TOKENHANDLER_PATH=$(find /home/ -name "tokenhandler.py" 2>/dev/null | head -n1)
```

Comment:
This find command can be improved in several ways. For example by searching in $HOME instead of /home/